internal/daysteps/daysteps.go

```diff
@@ -1,15 +1,44 @@
 package daysteps
 
 import (
+	"fmt"
+	"go1fl-4-sprint-final/internal/spentcalories"
+	"strconv"
+	"strings"
 	"time"
 )
 
 var (
 	StepLength = 0.65 // длина шага в метрах
 )
 
+// Функция parsePackage принимает строку с данными, которая содержит количество шагов
+// и продолжительность прогулки в формате 3h50m(3 часа 50 минут). Возвращает три значения:
+//
+// парсит строку формата "678,0h50m", где:
+// 678 — количество шагов.
+// 0h50m — продолжительность прогулки.
 func parsePackage(data string) (int, time.Duration, error) {
-	// ваш код ниже
+
+	strData := strings.Split(data, ",")
+	if len(strData) != 2 {
+		return 0, 0, fmt.Errorf("[parsePackage(data string)] неверный формат строки: %s", data)
+	}
+	step, err := strconv.Atoi(strData[0])
+	if err != nil {
+		return 0, 0, fmt.Errorf("[parsePackage(data string)] неверный формат количества шагов: %s", strData[0])
```

Comment:
Теряешь сообщение об ошибки
```suggestion
		return 0, 0, fmt.Errorf("[parsePackage(data string)] неверный формат количества шагов: %s: %w", strData[0], err)
```